code/parachain/runtime/primitives/src/topology.rs

```diff
@@ -132,10 +133,10 @@ impl Composable {
 				),
 				decimals: Some(6),
 				existential_deposit: 10_000,
-				ratio: Some(rational!(375 / 1_000_000)),
+				ratio: Some(rational!(375 / 1_000_000_000)),
```

Comment:
in picasso usdt id is 130 and parameters:
existential_deposit: 1500,
ratio: Some(rational!(15 / 1_000_000_000)),